README.md

```diff
@@ -61,6 +61,18 @@ Several higher-order Ruby projects have incorporated em-http and other Ruby HTTP
 - [em-eventsource](https://github.com/AF83/em-eventsource) - EventSource client for EventMachine
 - and many others.. drop me a link if you want yours included!
 
+## Release process
+
+(Added for Ably’s fork.)
+
+This library uses [semantic versioning](http://semver.org/). For each release, the following needs to be done:
+
+- Update the version number in [version.rb](./lib/em-http-request/version.rb) and commit the change.
+- Update `Changelog.md` with the changes since the previous release.
+- Create and push a tag named by prefixing the version number with the letter `v`; e.g. `git tag v1.1.8 && git push origin v1.1.8`.
```

Comment:
Are we going to release directly from `1.1.8` or will it be `1.0`. 
First tag/release starting from `1.1.8` will look a bit weird imo, wdyt

Reply:
Seems it's already released !